distribution/feature-packs/server-feature-pack/src/main/resources/content/bin/migrate-domain-clustered.cli

```diff
@@ -72,7 +72,7 @@ set persistenceProvider=jpa
 if (outcome == failed) of /profile=$clusteredProfile/subsystem=infinispan/cache-container=keycloak/distributed-cache=authorization/:read-resource
     # In migration from 3.0.0 to 3.2.0 there is authorization distributed-cache replaced with local-cache
     try
-        echo 
```

Comment:
Just double checking - this is intentional? I'm not sure an empty `echo` (without a space) is correct. Maybe it is - just double checking.

Reply:
Yes, it is intentional to remove whitespace noise, present in migration to previous versions, where necessary / appropriate.

For the format itself, the new entries follow the previous format (one empty `echo` / `echo` without args for better readability between entries).